model/app/konnector.go

```diff
@@ -217,7 +218,13 @@ func (m *KonnManifest) Create(db prefixer.Prefixer) error {
 	if err := couchdb.CreateNamedDocWithDB(db, m); err != nil {
 		return err
 	}
-	_, err := permission.CreateKonnectorSet(db, m.Slug(), m.Permissions())
+	// Add metadata
+	md, err := metadata.NewWithApp(m.DocSlug, m.Version(), permission.DocTypeVersion)
+	if err != nil {
+		return err
+	}
+
+	_, err = permission.CreateKonnectorSet(db, m.Slug(), m.Permissions(), md)
```

Comment:
I think it may be cleaner to pass `m.Version()` to `permission.CreateKonnectorSet`, and let `permission.CreateKonnectorSet` creates the `metadata.NewWithApp`.